tests/checks/indent.fish

```diff
@@ -487,6 +487,54 @@ level 2 } }
     # CHECK: {{^\}$}}
 } | $fish_indent
 
+echo 'false or {
+    echo hi
+}' | $fish_indent
+# CHECK: false or {
+# CHECK: {{^    }}echo hi
+# CHECK: {{^}}}
+
+echo 'true and {
+    echo hi
+}' | $fish_indent
+# CHECK: true and {
+# CHECK: {{^    }}echo hi
+# CHECK: {{^}}}
+
+echo 'test 1 -eq 1; or {
+    echo a
+    echo b
+}' | $fish_indent
+# CHECK: test 1 -eq 1; or {
+# CHECK: {{^    }}echo a
+# CHECK: {{^    }}echo b
+# CHECK: {{^}}}
+
+# Test single-line braces after conjunctions/operators stay on same line
+echo 'false and { echo yes }' | $fish_indent
+# CHECK: false and { echo yes }
+
+echo 'false or { echo yes }' | $fish_indent
+# CHECK: false or { echo yes }
```

Comment:
should add a semicolon before the and/or here else
they are basically the same as the first test above.